ansible/library/jcliff.py

```diff
@@ -40,21 +111,60 @@ def jcliff_absent(data=None):
    meta = {"absent": "not yet implemented"}
 
 def main():
-    default_jcliff_home = "/usr/share/jcliff-2.11.12/"
-    fields = {
-        "jcliff_home": {"default": default_jcliff_home, "type": "str" },
-        "jcliff": { "default": "/usr/bin/jcliff", "type": "str" },
-        "rules_dir": {"default": default_jcliff_home + "/rules", "type": "str"},
-        "wfly_home": {"required": True, "type": "str" },
-        "management_host": {"default": 'localhost', "type": "str" },
-        "management_port": {"default": '9990', "type": "str" },
-        "rule_file": { "required": True, "type": "str" },
-        "state": {
-            "default": "present",
-            "choices": ['present', 'absent'],
-            "type": 'str'
-        },
-    }
+    default_jcliff_home = "/usr/share/jcliff-2.11.12"
```

Comment:
This is not nice. It changes at every relase. Should we install jcliff to /usr/share/jcliff?

Reply:
It's an excellent point! I think I would keep the /usr/share/jcliff-<release>, so one can choose a specific version of jcliff if needed to, but I will add a symlink /usr/share/jcliff (pointing to the latest release).